### PR TITLE
feat: Add litmus test image with current PHP version (8.3)

### PR DESCRIPTION
**Pull request by susnux**
* For https://github.com/nextcloud/server/pull/44607#issuecomment-2031937115

### CODE REVIEW

**Review by come-nc — APPROVED** [review]
:elephant:

**Comment by st3iny** [comment]
Missing image `ghcr.io/nextcloud/continuous-integration-php8.3:latest` at https://github.com/nextcloud/docker-ci/pull/635

**Review by st3iny — APPROVED** [review]

**Comment by nickvergessen** [comment]
Will run the actions on the branch now

**Comment by nickvergessen** [comment]
Php-8.3 done, litmus-php8.3 done
Integration-php8.3 failed, i retried

File: integration-php8.3/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,7 @@
+FROM ghcr.io/nextcloud/continuous-integration-php8.3:latest
+
+RUN mkdir /tmp/server && \
+    cd /tmp/server && git clone --recursive https://github.com/nextcloud/server.git && \
+    cd /tmp/server/server/build/integration && composer install && \
+    cd /tmp/server/server/tests/acceptance && composer install && \
```
Comment (nickvergessen) [inline integration-php8.3/Dockerfile:6]: No longer exists

Comment (susnux) [reply integration-php8.3/Dockerfile]: Ah yes, fixed

File: litmus-php8.3/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,8 @@
+FROM ghcr.io/nextcloud/continuous-integration-php8.3:latest
+
+RUN apt-get update && apt-get install -y gcc coreutils make python && \
+    mkdir -p /tmp/litmus && \
+    wget -O /tmp/litmus/litmus-0.13.tar.gz http://www.webdav.org/neon/litmus/litmus-0.13.tar.gz && \
```
[the remark this answers is not in the document]
Comment (nickvergessen) [reply litmus-php8.3/Dockerfile:5]: Let's discuss in https://github.com/nextcloud/docker-ci/issues/788

**Pull request merged**